browser/ui/tabs/shared_pinned_tab_service.cc

```diff
@@ -712,15 +710,6 @@ void SharedPinnedTabService::MoveSharedWebContentsToBrowser(
     DCHECK(dummy_contents_data);
     dummy_contents_data->stop_propagation();
 
-    // Unfortunately, We can't replace existing tab contents with cached web
-    // contents. We should use restore method.
-    chrome::AddRestoredTabFromCache(
-        std::move(unique_shared_contents), browser, index,
-        /* group= */ {},
-        /* select */ !is_last_closing_browser, /* pin= */ true,
-        /* user_agent_override= */ {},
-        /* extra_data= */ {});
```

Comment:
It's okay to remove these as shared pinned tabs are experimental feature at the moment. I'm going to revisit this. Could you replace this with

```
TODO(sko) Should revisit this - https://github.com/brave/brave-core/pull/22936/commits/9fafe4d4a5cb82e37ecec3737f48a5a8f7490c0d#
NOTIMPLEMENTED()
```

Reply:
Would you mind then creating a follow-up issue to make sure this is documented?

Reply:
Not at all. Filed an issue https://github.com/brave/brave-browser/issues/37973